account_move_line_stock_info/README.rst

```diff
@@ -0,0 +1,84 @@
+============================
+Account Move Line Stock Move
```

Comment:
@bjeficent I think you have to run oca-gen-addon-readme

Reply:
You are right, my bad.